src/content/components/TimelineCanvas.js

```diff
@@ -138,7 +141,9 @@ export default class TimelineCanvas extends Component {
     });
 
     return <canvas className={className}
-                   ref='canvas'
+                   ref={canvas => {
+                     this._canvas = canvas;
+                   }}
```

Comment:
I'm going to fix that before merging. We've been bitten too much in the past with similar types of issues.